### PR TITLE
feat(metadata): Implement default labels 'framework' and 'language'

### PR DESCRIPTION
**Pull request by lpelypenko**
Hi to library contributors and maintenance,
Thank you very much for writing really nicely organized and tested typescript library and thank you for detailed `CONTRIBUTING.md` file.

**Objectives for this change**
Our team uses variety of different frameworks and they all use allure reporting. We want to index which test results are coming from which framework and all extensions that we use with allure (junit4, junit5, cypress, pytest) are adding 'framework' and 'language' into `the labels` array. Without it in TestCafe, our TestCafe tests are not indexed.

Example of how allure report looks like in junit5 allure extension:
```
{
  "attachments": [],
  "description": "",
  "fullName": "com.indeed.test.FooJUnit5Test.testPassedRetriedThreeTimes",
  "historyId": "1412e1206ef3da2fe04e10a6624ebaa8",
  "labels": [
    {
      "name": "host",
      "value": "XXXX"
    },
    {
      "name": "thread",
      "value": "59625@XXXXX.Test worker(12)"
    },
    {
      "name": "framework",
      "value": "junit-platform"
    },
    {
      "name": "language",
      "value": "java"
    },
    {
      "name": "package",
      "value": "com.indeed.test.FooJUnit5Test"
    },
    {
      "name": "testClass",
      "value": "com.indeed.test.FooJUnit5Test"
    },
    {
      "name": "testMethod",
      "value": "testPassedRetriedThreeTimes"
    },
    {
      "name": "suite",
      "value": "com.indeed.test.FooJUnit5Test"
    }
  ],
  "links": [],
  "name": "testPassedRetriedThreeTimes() (Repetition 3 of 3)",
  "parameters": [],
  "stage": "finished",
  "start": 1610591701664,
  "status": "skipped",
  "statusDetails": {
    "flaky": false,
    "known": false,
    "message": "Turn off the remaining repetitions as the test ultimately passed",
    "muted": false
  },
  "steps": [],
  "stop": 1610591701664,
  "uuid": "068b156d-df1b-460d-914e-85b2e2fc53da"
}
```

Example of current `testcafe-reporter-allure` report: 

```
{
    "uuid": "bf1d235f-a415-46f6-9f47-e5c4a45e6e34",
    "historyId": "f68ffef2-b7cc-442a-af80-0e2b4df2fffc",
    "status": "passed",
    "statusDetails": { "message": "", "trace": "" },
    "stage": "finished",
    "steps": [],
    "attachments": [],
    "parameters": [
        { "name": "accessibility", "value": "true" },
        { "name": "functional", "value": "false" },
    ],
    "labels": [
        { "name": "severity", "value": "Normal" },
        { "name": "suite", "value": "suite name" }
    ],
    "links": [],
    "start": 1625561608968,
    "name": "test",
    "fullName": "test",
    "stop": 1625561664345
}
```

I have reviewed `pytest` allure extension and it also produces `labels` with `framework` and `language`: 
```
"labels": [
    {
      "name": "parentSuite",
      "value": "tests"
    },
    {
      "name": "suite",
      "value": "test_example"
    },
    {
      "name": "subSuite",
      "value": "TestCase"
    },
    {
      "name": "host",
      "value": "IT-USA-VX0119"
    },
    {
      "name": "thread",
      "value": "17359-MainThread"
    },
    {
      "name": "framework",
      "value": "pytest"
    },
    {
      "name": "language",
      "value": "cpython3"
    },
    {
      "name": "package",
      "value": "tests.test_example"
    }
  ],
```

**Summary of changes**

-  Added two labels: `framework` and `languages` that will be always added to every test result
- Updated tests
- Updated `CONTRIBUTING.md` file

### CODE REVIEW

**Comment by lpelypenko** [comment]
Original discussion is in https://github.com/isaaceindhoven/testcafe-reporter-allure/pull/90

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ To ensure a smooth code review and merging process please check if the Pull Requ
 - The GitHub Action pipeline concludes successfully:
   - The code follows the style guide used within this project. This can be tested locally by running: `npm run test:lint`.
   - The code passes all unit tests. This can be tested locally by running: `npm run test:unit`.
-- The change does not break the example e2e tests. This can be tested locally by running `npm run test:e2e:allure`.
+- The change does not break the example e2e tests. This can be tested locally by running `npm run test:e2e`. After running e2e tests, `allure` directory will have `allure-restults` that can be used to see example allure report. Please run `npm run allure` and check how final report will look like
```
Comment (vanhoofmaarten) [inline CONTRIBUTING.md:36]: Spotted a tiny typo `allure-restults`. Can you edit this?

**Review by vanhoofmaarten — CHANGES_REQUESTED** [review]
Hey @lpelypenko,

Can you implement the requested change? Then we are all good to go!

File: CONTRIBUTING.md (hunk 1 above)
Comment (lpelypenko) [reply CONTRIBUTING.md]: Thank you very much for spotting it. Fixed!

**Review by vanhoofmaarten — APPROVED** [review]

**Comment by vanhoofmaarten** [comment]
Merged. Thank you for your contribution, @lpelypenko !

**Pull request merged**